### ISSUE TITLE
Toolbox requires thorough checks 

### ISSUE THREAD

**Issue by Kristin-2002** [ScotMarPhys/m_moorproc_toolbox#28]
Hi all,

I know a lot of you put a lot of work in the toolbox and tried your best to make it working. 

However, the last couple of weeks I had a brief look at the toolbox and found several issues without going in depth into the code:

- merge-conflicts that were pushed to remote without resolving (i search the entire toolbox and tried to resolve them all now)
- some scripts define variable which are then overwritten in scripts that are called (e.g. merging of mircrocat data still has this problem)
- the automatic meta-data propagation for oceansite netCDF files for CM and MicroCat was removed. I found a version were Yao contacts and his University was provided. This I found strange because he has nothing to do with the calibration of the instrument, etc.
- The guide for the toolbox is not up to date and difficult to follow which we already discussed in some meetings

**All in all I think the entire toolbox needs to be carefully checked, inconsistencies removed, certain functionalities restored**

That will take some time and patience. I am wondering whether it would be easier to start with an older version? 

@yfiring I saw that you used the version from DY120 for the current RAPID cruise. What is the reason for that?

Thank you
Kristin

**Comment by yfiring** [comment]
@Kristin-2002 I don't think we did use a DY120 version, as far as I can see dy174 derived from JC238-proc-RTmoor-noc which branched off from JC238 ... ?

Agreed we need to check the toolbox and documentation! As far as the variable overwriting I would suggest making more (most) things into functions to help avoid this.

**Comment by LewisDrysdale** [comment]
Sounds good to me. Very glad that there is interest in studying in detail and updating the toolbox. This is long overdue.

**Comment by Kristin-2002** [comment]
Hi all, that is great. Thanks for your quick and positive responses. 

I agree with the use of functions to avoid overwriting variables.

@yfiring 
My fault with DY174 – I did not noticed that I saw the oldest commits, not the newest commits in the commit history. The comments how I can see them have the JC238 commits included.

I will raise a new issue to establish a common branch strategy.
